### PR TITLE
Userfrienlyness

### PR DESCRIPTION
**Pull request by yohanboniface**
A bunch of cleaning after Zaatari experience.

### CODE REVIEW

File: kiwix.yml [hunk 1]
```diff
@@ -55,11 +51,10 @@ all:
     version: "2016-05-16"
     size: "2046925193"
     url: "http://download.kiwix.org/portable/wiktionary/kiwix-0.9+wiktionary_fr_all_2016-05.zip"
-    name: "Wiktionnaire -- Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire."
+    name: Wiktionnaire
     language: "fra"
-    description: "Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire."
+    description: Définition, traduction, prononciation, anagramme et synonyme sur le dictionnaire libre Wiktionnaire.
```
Comment (bochecha) [inline kiwix.yml:52]: Why not « Le dictionnaire libre » to make it more in line with other languages?

Not only that, but iirc the description is used for cards, where we'd prefer shorter strings?

Maybe we need a short and a long description?

File: kiwix.yml [hunk 2]
```diff
@@ -81,11 +75,10 @@ all:
     version: "2016-04-07"
     size: "437716601"
     url: "http://download.kiwix.org/portable/wikiversity/kiwix-0.9+wikiversity_fr_all_2016-04.zip"
-    name: "Wikiversité -- Une page de Wikiversité."
+    name: Wikiversité
     language: "fra"
     description: "Une page de Wikiversité."
```
Comment (bochecha) [inline kiwix.yml:74]: That doesn't seem like a great description?

File: kiwix.yml [hunk 3]
```diff
@@ -510,9 +510,9 @@ all:
     version: "2015-03-16"
     size: "2949838231"
     url: "http://download.kiwix.org/portable/other/kiwix-0.9+bil-tunisia_ar_all_2015-03.zip"
-    name: "Bil Tunisia -- Bil Tunisia videos"
+    name: "فيديو-بالتونسي"
     language: "ara"
-    description: "Bil Tunisia videos"
+    description: "مقاطع فيديو باللغه التونسيه"
```
Comment (letompouce) [inline kiwix.yml:437]: I was told the Bil Tunisia content is mostly in french, and not in arabic.

Comment (yohanboniface) [reply kiwix.yml:437]: True. It's half/half.
Like the same person doing half the phrase in French and half in Arabic…
But let's say this package is mainly targeted to people speaking both languages, so we can either keep the metadata in French (but then make it better) or in Arabic.

**Review by letompouce — COMMENTED** [review]
Removing 'title' and using 'name' for a title, still planning to turn from 'name' to 'title' does sound confusing for the people editing the yml.

What about dropping our custom 'name' attribute, which was made of 'title' + ' -- ' + 'name', and edit the 'title' attribute which comes straight from the Kiwix library.yml?

**Review by letompouce — APPROVED** [review]

**Comment by yohanboniface** [comment]
> What about dropping our custom 'name' attribute, which was made of 'title' + ' -- ' + 'name', and edit the 'title' attribute which comes straight from the Kiwix library.yml?

The catalog uses `name` at this stage.

**Review by letompouce — CHANGES_REQUESTED** [review]
You should apply this to the `kiwix.yml.sample` template as well.

File: kiwix.yml.sample [hunk 4]
```diff
@@ -10,4 +9,3 @@
     size: ""
```
Comment (letompouce) [inline kiwix.yml.sample]: r.e. a613635 : Kill the quotes! int ftw!

**Comment by letompouce** [comment]
r.e. 593452f & 5fd2ee5 we might want to bring these to upstream at some point. I agree with @yohanboniface that doing this right now would add some delay we cannot afford, still we should keep that in mind.

**Review by letompouce — APPROVED** [review]

**Pull request merged**